### ISSUE TITLE
Wiki Documentation

### ISSUE THREAD

**Issue by mike-ward** [mike-ward/Markdown-Edit#125]
I've started a [Wiki](https://github.com/mike-ward/Markdown-Edit/wiki) to document Markdown Edit. Please contribute if you can.

**Comment by mikjsmith** [comment]
Think Im being a numpty but.... how do I enter editing mode??

Saturday, February 27, 2016, 1:50:50 PM, you wrote:

MW> I've started a Wikito document Markdown Edit. Please contribute if you can.
MW> —
MW> Reply to this email directly or view it on GitHub.
MW>   
MW>      


---
Mike Smith

**Comment by mike-ward** [comment]
Hi,

You can't edit it directly. You have to clone it locally to your computer, make your changes and then submit a pull request.

It's not hard but it is odd and different if you've never done this kind of thing before.

**Comment by borekb** [comment]
Wikis have pull requests? How do I open one?

**Comment by mike-ward** [comment]
You clone like any other repository. The link is in the wiki sidebar.

![image](https://cloud.githubusercontent.com/assets/567927/13554466/76b46ec0-e376-11e5-9a9c-a7d7e980dcc0.png)

**Comment by borekb** [comment]
Yeah I know about cloning but how do I create a branch for the change and open a PR for it?

**Comment by mike-ward** [comment]
Yikes! Looks like you can't send PR's to wiki in GitHub. That's a surprise. I'll open up the wiki so others can edit.

**Comment by mike-ward** [comment]
OK, it's open to everyone now.

**Comment by borekb** [comment]
Yep, that's the simplest.

I've also seen a setup where the main wiki repo has a "sister" repo (full-fledged) where PRs are accepted and then synced back to the wiki. TypeScript team uses that:

- [the wiki](https://github.com/Microsoft/TypeScript/wiki) `https://github.com/Microsoft/TypeScript.wiki.git`
- [its sister repo](https://github.com/Microsoft/TypeScript-wiki) `https://github.com/Microsoft/TypeScript-wiki.git` – accepting PRs

**Comment by mike-ward** [comment]
I think it's weird that GitHub allows you clone it but not generate pull requests. After all, it's a repository
